src/instrumentations/grpc-js/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { isString, safeExecute } from '@lumigo/node-core/lib/utils';
+
+export const PAYLOAD_MAX_SIZE = 2048;
+
+export const concatenatePayload = (aggregatedData: string, currentData: unknown): string => {
+  return safeExecute(
+    () => {
+      if (aggregatedData.length >= PAYLOAD_MAX_SIZE) {
+        return aggregatedData;
```

Comment:
isn't it `<=` ?

Reply:
nope, we want to return the beginning of the aggregated data, so if it is long enough - we return it... 🤔 
Maybe I'm missing something. Can you look at the tests and give me an example of a new test that will fail?